modules/shapefile/src/lib/parsers/parse-shp-in-batches.js

```diff
@@ -0,0 +1,69 @@
+import BinaryAsyncIteratorReader from '../streaming/binary-async-iterator-reader';
```

Comment:
Will `BinaryAsyncIteratorReader` eventually move to `loader-utils`? It seems generalized